assets/css/app.css

```diff
@@ -68,3 +68,30 @@ h1 {
   border-color: #6c757d;
   color: #fff;
 }
+
+.explosion {
+  position: absolute;
+  width: 300px;
```

Comment:
Could we use `rem` instead? 🤔 If so, we just need to calculate it using our default font-size (I think that's 16 px), then it would be:

```
300 / 16 = 18.75rem
``` 

More info: 
https://engageinteractive.co.uk/blog/em-vs-rem-vs-px
https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9

Reply:
Not sure but maybe because of that: https://github.com/ricardoruwer/caiquenator/blob/master/assets/css/app.css#L10
We can set the width to `30rem`.